### ISSUE TITLE
`declarative_authorization.gemspec.src` and `rake build_gemspec` adds no value

### ISSUE THREAD

**Issue by timcharper** [stffn/declarative_authorization#50]
but it adds complexity and redundancy :( I was building my own version of the gem and didn't realize that the gemspec checked in to the repo was out of date.

Things work just fine if you stick the Dir.glob's right in the gemspec. The only reason I could perceive why you wouldn't want to do this if you used a gem build server that didn't trust you (like github, for example, which no longer builds gems). gem build gemspec will unfold the globs for you in the final product.

Patch: http://github.com/timcharper/declarative_authorization/tree/gemspec_fix

**Comment by stffn** [comment]
You are right.  It is still from the gems hosting at github.  I'll integrate your patch.

**Comment by timcharper** [comment]
I have branched from your master and cherry-picked the commit, so you can easily merge it in with:

```
git pull http://github.com/timcharper/declarative_authorization.git gemspec_fix
```

**Comment by stffn** [comment]
Fixed in c7f75e0370f179d96917
